Microsoft.Toolkit.Uwp.UI.Controls/ColorPickerButton/ColorPickerButtonSlider.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Foundation;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// A <see cref="Slider"/> implementation for use in the <see cref="ColorPickerButton"/>.
+    /// </summary>
+    /// <remarks>
+    ///
+    /// ColorPickerSlider is currently the same as Slider except it fixes a critical bug.
+    /// For details see:
+    ///
+    ///  * https://github.com/microsoft/microsoft-ui-xaml/issues/477
```

Comment:
We shouldn't be trying to have specific controls/work-arounds for platform issues, at least to this level of a custom control for a work-around. It sounds like this is fixed in the latest version of windows [as mentioned here](https://github.com/microsoft/microsoft-ui-xaml/issues/477#issuecomment-651207710), so we should rely on that fact and I can push for servicing of the fix down-level instead.

Reply:
If you can do that, that's great. There is another reason to create a new Slider control though. Some logic could be placed into it in the future as a simplification. For example -- background rendering. The slider could then have the color representation, color and channel given to it and it could calculate it's own background. That means we get rid of additional template parts and magic behind-the-scenes code to build background images. 

Also note that the existing Windows ColorPicker also has a ColorPickerSlider for similar reasons. When this goes into WinUI I expect both the ColorPicker and ColorPickerButton could share the same slider implementation.

Reply:
> I also noticed the value slide on the left was changing our selection of the color (unlike the built in picker). For instance if I start with green:

This is usually caused by constant accumulation of error between RGB/HSV. It is solved in the channels tab which coincidentally is the same code as the spectrum tab you have pictured... The solution is to always maintain an equivalent HSV color and modify that directly then convert to RGB instead of constantly RGB -> HSV -> RGB. I'll have to check this to see if the problem is in the color spectrum or I'm just not using the HSV color in this case.

Reply:
I was able to fix the Color Spectrum jumping all over the place when moving the third dimension slider. It was caused by conversion between RGB/HSV. I already had the infrastructure in place to maintain both color representations during editing (for the sliders tab) so I just now am always using the HSV color with the ColorSpectrum itself.

I'm still thinking it's better to break the Sliders into a separate 'ColorPickerButtonSlider' control that renders backgrounds itself. There would be a performance hit doing it this way but it may not be too bad and would simplify the overall structure of things. However, I would like to pursue that in a future update (it would be a minor breaking change). Otherwise, it will be a while before I get to it I think.